jax/lax/lax_control_flow.py

```diff
@@ -921,13 +924,18 @@ def scan(f, init, xs, length=None):
   if jax.api._jit_is_disabled():
     carry = init
     ys = []
+    if not forward:
```

Comment:
Would a backwards for loop work? (and then reverse `y` in Python before passing it to `stack`?)

Reply:
Yes that would work, and perhaps make debugging easier. I'll give it a shot.